src/environments/environment.prod.ts

```diff
@@ -9,22 +9,24 @@ export const environment = {
   fineractPlatformTenantId: window['env']['fineractPlatformTenantId'] || 'default',
   // For connecting to others servers running elsewhere update the base API URL
   baseApiUrls: window['env']['fineractApiUrls'] ||
-    'https://dev.mifos.io,https://demo.mifos.io,https://qa.mifos.io,https://staging.mifos.io,https://mobile.mifos.io,https://loans.test.oneacrefund.org,https://localhost:8443',
+    'https://loans.test.oneacrefund.org,https://loans.qa.oneacrefund.org,https://loans.oneacrefund.org',
   // For connecting to server running elsewhere set the base API URL
-  baseApiUrl: window['env']['fineractApiUrl'] || 'https://loans.test.oneacrefund.org',
+  baseApiUrl: window['env']['fineractApiUrl'] || 'https://loans.oneacrefund.org',
   allowServerSwitch: env.allow_switching_backend_instance,
   apiProvider: window['env']['apiProvider'] || '/fineract-provider/api',
   apiVersion: window['env']['apiVersion'] || '/v1',
   serverUrl: '',
   oauth: {
     enabled: true,  // For connecting to Mifos X using OAuth2 Authentication change the value to true
-    serverUrl: 'https://accounts.test.oneacrefund.org',
```

Comment:
We are still using test env for Keycloak since we are not yet live. Please revert